src/Infrastructure.VS.UnitTests/ActiveDocumentTrackerTests.cs

```diff
@@ -91,12 +93,12 @@ public void Dispose_ShouldUnregisterFromSelectionEvents()
         [TestMethod]
         public void Dispose_ShouldNoLongerRaiseEvents()
         {
-            var selectedFrame = CreateMockFrame(__WindowFrameTypeFlags.WINDOWFRAMETYPE_Document);
```

Comment:
The product code doesn't need to check the frame type property any more; we are only interested in SEID_DocumentFrame notifications, so we know that the frame must be a document frame,